content/admin/configuration/configuring-your-enterprise/configuring-rate-limits.md

```diff
@@ -61,3 +61,44 @@ You can apply Git rate limits per repository network or per user ID. Git rate li
+
+### About rate limits for {% data variables.product.prodname_actions %}
+
+Your {% data variables.product.product_name %} instance assigns each {% data variables.product.prodname_actions %} workflow job to a runner. If your instance cannot immediately assign a job to an available runner, the job will wait in a queue until a runner is available. If {% data variables.product.prodname_actions %} experiences sustained high load, the queue can back up, and the performance of {% data variables.product.product_location %} may degrade.
+
+To avoid this performance degradation, you can configure a rate limit for {% data variables.product.prodname_actions %}. This rate limit is expressed in job runs per minute. {% data variables.product.product_name %} calculates and applies the rate limit for the sum total of all job runs on the instance. If runs exceed the rate limit, additional runs will fail instead of entering the queue.
+
+![Screenshot of a rate-limited workflow run](/assets/images/enterprise/actions/actions-rate-limited.png)
+
+An appropriate rate limit protects {% data variables.product.product_location %} from abnormal usage of {% data variables.product.prodname_actions %} without interfering with day-to-day operations. The exact threshold depends on your instance's available resources and overall load profile. For more information about the hardware requirements for {% data variables.product.prodname_actions %}, see "[Getting started with {% data variables.product.prodname_actions %} for {% data variables.product.product_name %}](/admin/github-actions/getting-started-with-github-actions-for-your-enterprise/getting-started-with-github-actions-for-github-enterprise-server#review-hardware-requirements)."
+
+By default, the rate limit for {% data variables.product.prodname_actions %} is disabled. Because {% data variables.product.product_name %} can handle temporary spikes in usage without performance degradation, this rate limit is intended to protect against sustained high load. We recommend leaving the rate limit disabled unless you are experiencing performance problems. In some cases, {% data variables.contact.github_support %} may recommend that you enable a rate limit for {% data variables.product.prodname_actions %}. 
+
+### Enabling or disabling rate limits for {% data variables.product.prodname_actions %}
+
+{% data reusables.enterprise_installation.ssh-into-instance %}
+1. To enable and configure the rate limit, run the following two commands, replacing **RUNS-PER-MINUTE** with the value of your choice.
+
+   ```
+   ghe-config actions-rate-limiting.enabled true
+   ghe-config actions-rate-limiting.queue-runs-per-minute <em>RUNS-PER-MINUTE</em>
+   ```
```

Comment:
This code block needs to be specified as `shell`, otherwise the `<em>` appears literally:

![image](https://user-images.githubusercontent.com/4434330/163904098-b7e08668-9c5f-4ae0-b55d-03b5183c4e64.png)

````suggestion
   ```shell
   ghe-config actions-rate-limiting.enabled true
   ghe-config actions-rate-limiting.queue-runs-per-minute <em>RUNS-PER-MINUTE</em>
   ```
````